crates/project_model/src/cargo_workspace.rs

```diff
@@ -64,6 +64,9 @@ pub struct CargoConfig {
 
     /// rustc target
     pub target: Option<String>,
+
+    /// rustc private crate source
+    pub rustc_source: Option<AbsPathBuf>,
```

Comment:
Does this need to point to the full fork, `compiler/` or `compiler/rustc_middle`?

Reply:
I've only tested this by pointing to the full fork, regardless anywhere in the `rustc` tree currently produces the same result since we end up calling `cargo metadata` which gives results for the _whole_ workspace.